iree/turbine/kernel/wave/iree_utils.py

```diff
@@ -208,4 +208,5 @@ def generate_iree_ref(
         kernel_outputs,
         run=True,
         run_bench=kwargs.get("run_bench", False),
+        pass_outputs_to_kernel=False,
```

Comment:
Semantically, it should be named `(not)inplace_outputs`, but we are using `inplace=False` only in `generate_iree_ref` so I think we can just unify those 2 params. Also, while we are here, can you pass `inplace=False` explicitly in `generate_iree_ref`.

Reply:
Correct me if I'm wrong but looks like inplace is always False. It has not been turned on yet. So I don't think we can unify them unless I turn it on?

Reply:
Sorry my mistake, yes I will unify and change.

Reply:
`test_execute` uses `inplace=True`